crmsh/migration-supported-resource-agents.txt

```diff
@@ -0,0 +1,144 @@
+ocf:heartbeat:WAS
+ocf:heartbeat:WAS6
+ocf:heartbeat:Xinetd
+ocf:heartbeat:aliyun-vpc-move-ip
+ocf:heartbeat:apache
+ocf:heartbeat:aws-vpc-move-ip
+ocf:heartbeat:aws-vpc-route53
+ocf:heartbeat:awseip
+ocf:heartbeat:awsvip
+ocf:heartbeat:azure-events
+ocf:heartbeat:azure-events-az
+ocf:heartbeat:azure-lb
+ocf:heartbeat:conntrackd
+ocf:heartbeat:corosync-qnetd
+ocf:heartbeat:crypt
+ocf:heartbeat:db2
+ocf:heartbeat:dhcpd
+ocf:heartbeat:docker
+ocf:heartbeat:docker-compose
+ocf:heartbeat:dummypy
+ocf:heartbeat:ethmonitor
+ocf:heartbeat:exportfs
+ocf:heartbeat:galera
+ocf:heartbeat:garbd
+ocf:heartbeat:gcp-ilb
+ocf:heartbeat:gcp-pd-move
+ocf:heartbeat:gcp-vpc-move-ip
+ocf:heartbeat:gcp-vpc-move-vip
+ocf:heartbeat:iSCSILogicalUnit
+ocf:heartbeat:iSCSITarget
+ocf:heartbeat:iface-bridge
+ocf:heartbeat:iface-macvlan
+ocf:heartbeat:iface-vlan
+ocf:heartbeat:ldirectord
+ocf:heartbeat:lvmlockd
+ocf:heartbeat:mariadb
+ocf:heartbeat:mdraid
+ocf:heartbeat:mpathpersist
+ocf:heartbeat:mysql
+ocf:heartbeat:mysql-proxy
+ocf:heartbeat:named
+ocf:heartbeat:nfsnotify
+ocf:heartbeat:nfsserver
+ocf:heartbeat:nginx
+ocf:heartbeat:nvmet-namespace
+ocf:heartbeat:nvmet-port
+ocf:heartbeat:nvmet-subsystem
+ocf:heartbeat:oraasm
+ocf:heartbeat:oracle
+ocf:heartbeat:oralsnr
+ocf:heartbeat:osceip
+ocf:heartbeat:ovsmonitor
+ocf:heartbeat:pgagent
+ocf:heartbeat:pgsql
+ocf:heartbeat:podman
+ocf:heartbeat:portblock
+ocf:heartbeat:postfix
+ocf:heartbeat:powervs-subnet
+ocf:heartbeat:rabbitmq-cluster
+ocf:heartbeat:rabbitmq-server-ha
+ocf:heartbeat:redis
+ocf:heartbeat:rsyncd
+ocf:heartbeat:sfex
+ocf:heartbeat:sg_persist
+ocf:heartbeat:slapd
+ocf:heartbeat:storage-mon
+ocf:heartbeat:symlink
+ocf:heartbeat:tomcat
+ocf:suse:aws-vpc-move-ip
+stonith:fence_aliyun
+stonith:fence_alom
+stonith:fence_apc
+stonith:fence_apc-snmp
+stonith:fence_aws
+stonith:fence_azure-arm
+stonith:fence_bladecenter
+stonith:fence_brocade
+stonith:fence_cisco-mds
+stonith:fence_cisco-ucs
+stonith:fence_compute
+stonith:fence_docker
+stonith:fence_drac5
+stonith:fence_eaton-snmp
+stonith:fence_eaton-ssh
+stonith:fence_emerson
+stonith:fence_eps
+stonith:fence_gce
+stonith:fence_hds-cb
+stonith:fence_hpblade
+stonith:fence_ibm-powervs
+stonith:fence_ibm-vpc
+stonith:fence_ibmblade
+stonith:fence_ibmz
+stonith:fence_ifmib
+stonith:fence_ilo-moonshot
+stonith:fence_ilo-mp
+stonith:fence_ilo-ssh
+stonith:fence_ilo2
+stonith:fence_intelmodular
+stonith:fence_ipdu
+stonith:fence_ipmilan
+stonith:fence_ironic
+stonith:fence_kdump
+stonith:fence_ldom
+stonith:fence_lpar
+stonith:fence_mpath
+stonith:fence_netio
+stonith:fence_openstack
+stonith:fence_pve
+stonith:fence_raritan
+stonith:fence_rcd-serial
+stonith:fence_redfish
+stonith:fence_rhevm
+stonith:fence_rsa
+stonith:fence_rsb
+stonith:fence_sanbox2
+stonith:fence_sbd
+stonith:fence_scsi
+stonith:fence_vbox
+stonith:fence_virsh
+stonith:fence_vmware
+stonith:fence_vmware-rest
+stonith:fence_wti
+stonith:fence_xenapi
+stonith:fence_zvm
```

Comment:
How has the above data been generated?
Can we automatically do that based on the local installed resource-agents and fence-agents-* rpm?

Reply:
> How has the above data been generated?

Extracted the file list from resource-agents/fence-agents.

> Can we automatically do that based on the locally installed resource-agents and fence-agents-* rpm?

No, because we need to know what is supported by SLES 16 instead of SLES 15 SP6.